app/controllers/api/v1/authentication_controller.rb

```diff
@@ -195,9 +212,9 @@ def current_user_channels
       # to successfully authenticate. This helps prevent man-in-the-middle attacks by ensuring that only this
       # client that can authenticate the OTP code with it.
       # Note: if user is nil, we generate a fake token so as to ward off unruly hackers.
-      def otp_auth_key_for(user)
-        if user.present?
-          user.most_recent_token.otp_auth_key
+      def otp_auth_key_for_user
```

Comment:
I'm not a huge fan of this change... we've swapped something explicit for something implicit... the new change relies on there being an `@user` variable whereas the previous method allowed the calling method to define it. Minor tweak but I think the previous was more flexible.

Reply:
Makes sense. We will make it as previous one. :)